src/browser/register.ts

```diff
@@ -4,12 +4,12 @@ import "./pages/error.css"
 import "./pages/global.css"
 import "./pages/login.css"
 
-async function registerServiceWorker(): Promise<void> {
+export async function registerServiceWorker(): Promise<void> {
   const options = getOptions()
   const path = normalize(`${options.csStaticBase}/dist/serviceWorker.js`)
   try {
     await navigator.serviceWorker.register(path, {
-      scope: (options.base ?? "") + "/",
+      scope: options.base + "/",
```

Comment:
TLDR: `options.base` will always return a string. No need to use [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)
See comment message for explanation: https://github.com/cdr/code-server/pull/3200/commits/67dbb36732afe76c26c4d7db793c5912a1dfafcb